src/styles/blocks/contact-us.scss

```diff
@@ -0,0 +1,165 @@
+.contact-us {
+  &__title {
+    margin-bottom: 48px;
+
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 32px;
+    letter-spacing: -1.5px;
+    text-align: center;
+
+    grid-column: 1 / -1;
+
+    @include on-tablet {
+      font-size: 48px;
+      line-height: 48px;
+
+      grid-column: 2 / 6;
+    }
+
+    @include on-desktop {
+      grid-column: 5 / 9;
+    }
+
+    &-wrapper {
```

Comment:
`contact-us__title-wrapper` is the other element so don't insert into `contact-us__title`